updated_main.py

```diff
@@ -3,57 +3,89 @@
 import bs4
 import time
 import json
+import os
+from email.message import EmailMessage
+import ssl
+import smtplib
 
-# Running a for loop to get the latest price of the GBP
-for i in range(100):
-
-    # Now extracting the value of British Pound in Indian Rupees
-    currency = "convert 1 pound to rs"
-
-    url = "https://google.com/search?q=" + currency
 
-    request_result = requests.get(url)
+def getting_current_price():
```

Comment:
```suggestion
def get_current_price():
```
Variable names should be imperative.

Reply:
This is not done yet.

Reply:
Done now